go/vt/vtgate/evalengine/expr_bvar.go

```diff
@@ -26,12 +26,14 @@ import (
 
 type (
 	BindVariable struct {
-		Key       string
-		Type      sqltypes.Type
-		Collation collations.TypedCollation
+		Key               string
+		Type              sqltypes.Type
+		Collation         collations.ID
+		dynamicTypeOffset int
```

Comment:
maybe a little comment explaining this field?

Reply:
Done :+1: